index.html

```diff
@@ -6,176 +6,180 @@
-                                    <div class="list-4">JavaScript</div>
-                                </a>
-                            </li>
-                            <li><a href="index.html"  class="list-2">
-                                <div class="list-3"><img src="image/Ellipse-3.png" alt="social1" /></div>
-                                <div class="list-4">HTML</div>
-                                </a>
-                            </li>
-                            <li><a href="index.html"  class="list-2">
-                                <div class="list-3"><img src="image/Ellipse-2.png" alt="social1" /></div>
-                                <div class="list-4">css</div>
-                                </a>
-                            </li>
-                        <li class="list">Frameworks</li>
-                        <li class="list">Skills</li>
+                <section class="grid-item-section">
+                    <h1 class="title">Multi-Post Stories</h1>
+                    <ul class="sub-title">
+                        <li>CANOPY</li>
+                        <li>Back End Dev</li>
+                        <li>2015</li>
                     </ul>
-            </section>
-            <section class="contact-me">
-                    <h1 class="contact-title">Contact me</h1>
-                    <p class="content-3">
-                        If you have an application you are interested in developing, 
-                        a feature that you need built or a project that needs coding. 
-                        i'd love to help with it.</p>
-                <form action="https://formspree.io/f/{form_id}" methop="post" class="text-me">
-                    <input type="text" class="simple" placeholder="User name" maxlength="30" required>
-                    <input type="email" class="simple" placeholder="Email" maxlength="30" required>
-                    <textarea class="text-area" placeholder="comment" maxlength="500" required></textarea>
-                    <button type="submit" class="project">Get in touch</button>
-                </form>       
-            </section>
+                    <p>
+                        A daily selection of privately personalized reads; no account or sign-ups required.</p>
+                    <ul class="category">
+                        <li class="category-list"><a href="index.html">html</a></li>
+                        <li class="category-list"><a href="index.html">css</a></li>
+                        <li class="category-list"><a href="index.html">javaScript</a></li>
+                    </ul>
+                    <button type="submit" class="project">See Project</button>
+                </section>
+            </div>
         </div>
+        <section class="about">
+            <div class="about-content1">
+                <h1>About Myself</h1>
+                <p>
```

Comment:
- Please work on the margin-top of the `about section` to the `project section`, so that it looks like that of the Figma design with a good UI 
- Please work on the left and right margin on the `project section` as it is too large on your design
##### Your design (with directions on areas to work on)
![Screenshot from 2022-06-10 08-38-17](https://user-images.githubusercontent.com/58771507/173015790-48335e6e-67fd-4950-963c-aa199644aba8.png)

##### Figma design 
![Screenshot from 2022-06-10 08-41-59](https://user-images.githubusercontent.com/58771507/173015995-bbb5ab29-079e-4559-8c9e-58dd5dca0a69.png)